core/pom.xml

```diff
@@ -16,6 +16,7 @@
     <description>InputFormat/OutputFormat to integrate with Hadoop Map Reduce and supported tools</description>
```

Comment:
Commit should be called "Upgrade to Infinispan 9.4.1.Final", as "backport prod changes" does not mean anything :)